src/pages/home/index.jsx

```diff
@@ -57,7 +57,20 @@ export function App() {
   }
 
   function notifyFailed() {
-    toast.success("o produto não foi deletetado!", {
+    toast.success("o produto não foi deletado!", {
```

Comment:
Se não for seguir o que eu passei ler a documentação do toast, você ta passando um sucesso quando era para ser um error como `toast.error` e não `toast.success`